bigtable-hbase-replication/pom.xml

```diff
@@ -201,12 +201,6 @@
       <version>3.8.0</version>
       <scope>test</scope>
     </dependency>
-    <!--    <dependency>-->
```

Comment:
We should also remove the dependency on the HBase-server, if not in this PR then in the next one.

Reply:
Will do it in next PR. remove ReplicationSource